code/__DEFINES/colors.dm

```diff
@@ -247,7 +247,24 @@
 #define CIRCUIT_COLOR_MEDICAL "#00CCFF"
 #define CIRCUIT_COLOR_ENGINEERING "#F8D700"
 #define CIRCUIT_COLOR_SUPPLY "#C47749"
-
+#define CABLE_COLOR_BLUE 	"blue"
```

Comment:
Why is this file changed at all? It's just moving stuff.

Reply:
it's just moving color defines next to each other there is no issue here